src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,209 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+    private boolean end;
```

Comment:
Would it be better if it was `isEnd`? For making a boolean variable "sounds like" boolean.